app/views/trips/_import_form.html.haml

```diff
@@ -0,0 +1,6 @@
+=form_for @trip, action: "import", method: "post", url: "" do |f|
+  =f.label :Insert_Student_OSISES_Below
+  =f.submit "Import Students", :class => "btn.btn-primary.a-white"
+  %br
+  =f.text_area :importStudents, placeholder: "Enter Student Osises Here:", size:"100"
```

Comment:
Should use `snake_case` in general, so `import_students` would be generally expected here